i-slide.js

```diff
@@ -399,16 +404,27 @@ class ISlide extends HTMLElement {
     const width = this.getAttribute('width');
     const type = this.getAttribute('type');
 
-    if ((this.src !== src) || (this.type !== type)) {
+    if ((this.src !== src) || (this.type !== type) ||
+        (this.width !== width && this.busyCounter > 0)) {
```

Comment:
I'm not sure I understand the logic here (probably because I shouldn't review this while doing something else), but at the very least I think it should have better comments, and maybe specific associated tests

Reply:
I agree tests would be useful but I don't have any immediate easy solution to test this bit of code, because that also requires intercepting and delaying network operations to create the right conditions for the function to be called more than once before it is done with its fetch-and-render loop. I added some comments instead.